std/assembly/index.d.ts

```diff
@@ -116,6 +116,8 @@ declare function offsetof<T>(): usize;
 declare function offsetof<T>(fieldName: keyof T | string): usize;
 /** Determines the offset of the specified field within the given class type. Returns the class type's end offset if field name has been omitted. Compiles to a constant. */
 declare function offsetof<T>(fieldName?: string): usize;
+/** Determine the name of a given class type instance. */
```

Comment:
Perhaps "Determines the name of the given type." since it also covers basic types.

Reply:
Reasonable and easier to understand.